isis/CMakeLists.txt

```diff
@@ -28,15 +30,15 @@ set(PACKAGE            "ISIS")
 set(PACKAGE_NAME       "USGS ISIS")
 
 # Version number
-set(VERSION            "3.5.00.0")
+set(VERSION            "3.6.00.0")
```

Comment:
Does this version number not look a little off...?

Reply:
We're on 3.6.0 now right? This just has a bunch of extra 0s for minor versions.

Reply:
Ian and Makayla updated this when they put in gtest.

Reply:
In the future we should make these kinds of changes on dev and then merge them into feature branches.